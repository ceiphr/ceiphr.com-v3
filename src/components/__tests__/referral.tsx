import React from "react"
import renderer from "react-test-renderer"

import Referral from "../referral"

describe("Referral", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Referral />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
