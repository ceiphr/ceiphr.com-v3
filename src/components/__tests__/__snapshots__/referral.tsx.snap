// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Referral renders correctly 1`] = `
<div
  className="card referral"
>
  <a
    aria-hidden="true"
    href="https://m.do.co/c/b95c2a8a5568"
    rel="noopener"
    target="_blank"
  >
    <div
      className="referral__logo"
    >
      <test-file-stub
        alt="DigitalOcean"
      />
       
    </div>
    <p>
      Helping millions of developers easily build, test, manage, and scale applications of any size.
    </p>
  </a>
</div>
`;
